ql/src/test/queries/clientpositive/dbtxnmgr_showlocks.q

```diff
@@ -1,3 +1,4 @@
+ --! qt:replace:/(initiated\s+---\s+---\s+)[0-9]*(\s+---)/$1#Masked#$2/
```

Comment:
Nit: maybe add a comment here about what's going on, to save someone time down the road

Reply:
Done